meson.build

```diff
@@ -483,17 +547,16 @@ if get_option('enable-app')
   if get_option('platform') == 'android'
     warning('android app is not supported for now, building app skipped')
   else
-    # this is needed for reinforcement application. We can move this to reinforecement app dependency
-    jsoncpp_dep = dependency('jsoncpp') # jsoncpp
-    libcurl_dep = dependency('libcurl')
-    if not tflite_dep.found()
-      error('Tensorflow-Lite dependency not found')
+    if host_machine.system() != 'windows'
+      if not tflite_dep.found()
+        error('Tensorflow-Lite dependency not found')
+      endif
     endif
     subdir('Applications')
   endif
 endif
 
-if get_option('platform') != 'android'
+if get_option('platform') != 'android' and host_machine.system() != 'windows'
```

Comment:
We should use get_option() here really, since it's already optional by `meson.options`